integrations/dd-sdk-android-okhttp-otel/README.md

```diff
@@ -0,0 +1,7 @@
+# Datadog SDK OpenTelemetry extensions for OkHttp
+
+See the dedicated [Datadog SDK instrumentation for OkHttp documentation][1] to learn how track network requests made by the `OkHttp` library automatically.
+See the dedicated [Datadog SDK Android for OpenTelemetry documentation][2] to learn how to add a parent span to network requests made by the `OkHttp` library.
+
+[1]: https://docs.datadoghq.com/real_user_monitoring/android/advanced_configuration/#automatically-track-network-requests
+[2]: https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/android?tab=kotlin
```

Comment:
Is the second link expected to work yet? It doesn't currently but I couldn't find an alternative, guessing these docs just aren't published yet.

Reply:
yes the idea is to have this merge and release in the next version and then merge the documentation PR which is already opened.